specs/order_spec.rb

```diff
@@ -78,33 +80,43 @@
   end
 end
 
-# TODO: change 'xdescribe' to 'describe' to run these tests
-xdescribe "Order Wave 2" do
+
+describe "Order Wave 2" do
   describe "Order.all" do
-    it "Returns an array of all orders" do
-      # TODO: Your test code here!
-    end
+    it "Returns an array of all orders" do # DONE
+      Grocery::Order.all.must_be_kind_of Array
+    end # it/do
 
-    it "Returns accurate information about the first order" do
-      # TODO: Your test code here!
-    end
 
-    it "Returns accurate information about the last order" do
-      # TODO: Your test code here!
-    end
-  end
+    it "Returns accurate information about the first order" do # DONE
+      Grocery::Order.all[0].id.must_equal 1
+      product_in_1st = {"Slivered Almonds" => 22.88, "Wholewheat flour" => 1.93, "Grape Seed Oil" => 74.9}
+      Grocery::Order.all[0].products.must_equal product_in_1st
```

Comment:
Instead of calling `.all` multiple times, you should call it once and store the result in a variable (the _act_ step) and then use that variable for your checks (_assert_ step). This will make the tests a little more readable.